src/framework/preferences/js/LetterSpaceSchemas.js

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_3_0_0 = fluid_3_0_0 || {};
+
+(function (fluid) {
+    "use strict";
+
+    /*******************************************************************************
+    * Starter auxiliary schema grade
+    *
+    * Contains the settings for 7 preferences: text size, line space, text font,
+    * contrast, table of contents, inputs larger and emphasize links
+    *******************************************************************************/
+
+    // Fine-tune the starter aux schema and add speak panel
+    fluid.defaults("fluid.prefs.auxSchema.letterSpace", {
+        gradeNames: ["fluid.prefs.auxSchema"],
+        auxiliarySchema: {
+            "namespace": "fluid.prefs.constructed",
```

Comment:
Line 28-34 are common options at creating a new aux schema for a preference. Would be easier to reuse by extracting them into a base grade.